### PR TITLE
[feature](catalog) support clob type in oracle catalog

### PR DESCRIPTION
**Pull request by lsy3993**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by zy-kkk — COMMENTED** [review]
add test case

File: fe/be-java-extensions/jdbc-scanner/src/main/java/org/apache/doris/jdbc/JdbcExecutor.java [hunk 1]
```diff
@@ -1482,6 +1482,53 @@ private void ipPutToString(Object[] column, boolean isNullable, int numRows, lon
         UdfUtils.copyMemory(bytes, UdfUtils.BYTE_ARRAY_OFFSET, null, bytesAddr, offsets[numRows - 1]);
     }
 
+    private void clobToString(Object[] column, boolean isNullable, int numRows, long nullMapAddr,
```
[the remark this answers is not in the document]
Comment (lsy3993) [reply fe/be-java-extensions/jdbc-scanner/src/main/java/org/apache/doris/jdbc/JdbcExecutor.java]: done

**Review by zy-kkk — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.17 seconds
 stream load tsv:          510 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          30.2 seconds inserted 10000000 Rows, about 331K ops/s
 storage size: 17160758487 Bytes

**Review by LemonLiTree — APPROVED** [review]
LGTM

**Pull request merged**